app/Services/Model/Imp/TrackChairRankingService.php

```diff
@@ -0,0 +1,228 @@
+
+            $track_chair_rating_type = $selection_plan->getTrackChairRatingTypeById($track_chair_rating_type_id);
+            if (is_null($track_chair_rating_type))
+                throw new EntityNotFoundException("Track chair rating type not found.");
+
+            if ($track_chair_rating_type->getScoreTypes()->count() > 0)
+                throw new ValidationException("There is at least one score type associated to this rating type.");
+
+            $selection_plan->removeTrackChairRatingType($track_chair_rating_type);
+        });
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function addTrackChairScoreType(
+        SelectionPlan $selection_plan, int $track_chair_rating_type_id, array $payload): PresentationTrackChairScoreType
+    {
+        return $this->tx_service->transaction(function () use ($selection_plan, $track_chair_rating_type_id,  $payload)
+        {
+            $current_member = $this->resource_server_ctx->getCurrentUser();
+            $this->checkPermissions($selection_plan->getSummit(), $current_member);
+
+            $track_chair_rating_type = $selection_plan->getTrackChairRatingTypeById($track_chair_rating_type_id);
+            if (is_null($track_chair_rating_type))
+                throw new EntityNotFoundException("Track chair rating type not found.");
+
+            $track_chair_score_type = new PresentationTrackChairScoreType();
+            $track_chair_score_type->setScore(intval($payload['score']));
+            $track_chair_score_type->setName($payload['name']);
+            $track_chair_score_type->setDescription($payload['description']);
+
+            $track_chair_rating_type->addScoreType($track_chair_score_type);
+
+            return $track_chair_score_type;
+        });
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function updateTrackChairScoreType(
+        SelectionPlan $selection_plan, int $track_chair_rating_type_id, int $track_chair_score_type_id, array $payload): PresentationTrackChairScoreType
+    {
+        return $this->tx_service->transaction(function () use ($selection_plan, $track_chair_rating_type_id, $track_chair_score_type_id, $payload)
+        {
+            $current_member = $this->resource_server_ctx->getCurrentUser();
+            $this->checkPermissions($selection_plan->getSummit(), $current_member);
+
+            $track_chair_rating_type = $selection_plan->getTrackChairRatingTypeById($track_chair_rating_type_id);
+            if (is_null($track_chair_rating_type))
+                throw new EntityNotFoundException("Track chair rating type not found.");
+
+            $track_chair_score_type = $track_chair_rating_type->getScoreTypeById($track_chair_score_type_id);
+
+            if (is_null($track_chair_score_type))
+                throw new EntityNotFoundException("Track chair score type not found.");
+
+            $track_chair_score_type->setScore(intval($payload['score']));
```

Comment:
score is not valid check is an order too , so check here
https://github.com/OpenStackweb/summit-api/blob/658d1d0235473f845e86db65ae501326c39ef955/app/Services/Model/Imp/ExtraQuestionTypeService.php#L103

Reply:
done